### ISSUE TITLE
RCS1192 is incorrectly reported

### ISSUE THREAD

**Issue by DaniilPalii** [JosefPihrt/Roslynator#701]
**Product and Version Used**:
Roslynator.Analyzers 2.3.0

**Code**: 
```csharp
var a = @".\some";
```

**Description**:
"RCS1192: Unnecessary usage of verbatim string literal" is reported in the above line but there is a backslash and there is sense to use verbatim string to avoid escaping.

**Comment by JosefPihrt** [comment]
I cannot reproduce this bug. I checked exact commit that the 2.3.0 was built from:

https://github.com/JosefPihrt/Roslynator/blob/cb44180d091eca3a0ffc45f9dda097f0e50320da/src/Analyzers/CSharp/Analysis/UseRegularStringLiteralInsteadOfVerbatimStringLiteralAnalyzer.cs#L51

You can see that RCS1192 is not reported if the text contains backslash.
